language/examples/document-summarization/summarization_with_docai.ipynb

```diff
@@ -0,0 +1,731 @@
+{
```

Comment:
<p>"a reduce function that concatenate" --&gt; "a reduce function that concatenate<strong><u>s</u></strong>"</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/generative-ai/pull/131/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='32'/>